Dockerfile

```diff
@@ -32,7 +32,7 @@ RUN mkdir -p /usr/local/lib/bats/bats-mock \
 RUN if [[ -e /bin/bash ]]; then echo "/bin/bash already exists"; exit 1; else ln -s /usr/local/bin/bash /bin/bash; fi
 
 # Expose BATS_PATH so people can easily use load.bash
-ENV BATS_PATH=/usr/local/lib/bats
```

Comment:
Does the new image define a `BATS_PATH`? I couldn't spot it with this quick check:

```bash
$ docker run -it --rm --entrypoint bash bats/bats:latest@sha256:97d91ee0aa9771e696cdf44c2b1672af484fd846eaf52ba2db6061f5c78a89d5
bash-4.4# echo $BATS_PATH

bash-4.4# exit
exit
$ docker run -it --rm --entrypoint bash bats/bats:1.2.1@sha256:2defd45426be3b5bc7415455ca529c910dab7473717eeece9c3cc26efa5ca037
bash-5.0# echo $BATS_PATH

bash-5.0# exit
```

Reply:
Full disclosure, it was a massive assumption on my part because my var was the wrong value. I'll dig deeper.

Reply:
Yeah, the path location definitely seems to have changed! Perhaps we just need to update the value to the new path, rather than rename it? I was wondering if people even need to the `load` line in the usage example in the readme anymore.

Reply:
Oh, I see that `BATS_PATH` is a think that exists in 1.2.x now, and is set up at run-time when the `bats` command is executed:
https://github.com/bats-core/bats-core/blob/5ed2fb0addf99b402ee2b2e7c3ca783350d4073d/bin/bats#L54

Dang!

Reply:
Yeah, which is why it's overriding the var we set. Bit of a pain.

I tried running without that line and it doesn't load the extensions. That's what bats-support does to my understanding and why it needs to be called explicitly.

Beyond putting the extensions in a fairly non-standard place (`/opt/bats/bin/bats`), I think we're suck needing a new var. Thoughts?

Reply:
And this is where I regret that we never versioned the Docker image, and so I'm scared of removing the var and breaking everyone's plugin tests. Maybe that's inevitable.

Luckily https://buildkite.com/docs/plugins/writing doesn't mention it, it seems only to be the usage instructions in the README here.

Given it's a breaking change, should we bump the versions of bats-assert, bats-mock, etc too? 🤔

Reply:
@blaknite Please can you share your thoughts on bump the versions of bats-assert, bats-mock etc ?

Reply:
[bats-assert](https://github.com/ztombol/bats-assert) latest version is v0.3.0 and [bats-mock](https://github.com/lox/bats-mock) latest version is v1.3.0

From [dockerfile](https://github.com/buildkite-plugins/buildkite-plugin-tester/blob/master/Dockerfile), already using these latest versions